requirements.txt

```diff
@@ -0,0 +1,26 @@
+# Dev packages
+--editable git+https://github.com/inducer/pymbolic.git#egg=pymbolic
+--editable git+https://github.com/inducer/islpy.git#egg=islpy
+--editable git+https://github.com/inducer/pyopencl.git#egg=pyopencl
+--editable git+https://github.com/inducer/loopy.git#egg=loo.py
+--editable git+https://github.com/inducer/dagrt.git#egg=dagrt
+--editable git+https://github.com/inducer/leap.git#egg=leap
+--editable git+https://github.com/inducer/modepy.git#egg=modepy
+--editable git+https://github.com/inducer/meshmode.git#egg=meshmode
+--editable git+https://github.com/inducer/grudge.git#egg=grudge
+
+# Toy Fortran parser for Loopy
+#
+# Unfortunately, f2py doesn't exist as a package on the package index:
+# https://pypi.org/search/?q=f2py
+#
+# To explain the context of this a bit: f2py is a Fortran->Python wrapper generator that ships
+# with numpy. Due to limitations of the old code base (no F90), Pearu (Peterson, original author of
+# f2py) decided to restart from scratch, starting with a Fortran parser. That's as far as he got,
+# and a forked version of this is in this repo.
+
+git+https://github.com/inducer/f2py.git#egg=fparser
```

Comment:
These don't get pulled it automatically, I'm guessing because pip looks at setup.py instead of requirements.txt for the dependent packages. Same for mpi4py etc. below.